src/farm/utils.ts

```diff
@@ -213,3 +213,60 @@ export function stringifyQuery(query: [string, string][]): string {
 export interface JsPluginExtended extends JsPlugin {
   [key: string]: any
 }
+
+export const CSS_LANGS_RES: [RegExp, string][] = [
+  [/\.(less)(?:$|\?)/, 'less'],
+  [/\.(scss|sass)(?:$|\?)/, 'sass'],
+  [/\.(styl|stylus)(?:$|\?)/, 'stylus'],
+  [/\.(css)(?:$|\?)/, 'css'],
+]
+
+export const JS_LANGS_RES: [RegExp, string][] = [
+  [/\.(js|mjs|cjs|)(?:$|\?)/, 'js'],
```

Comment:
```suggestion
  [/\.(js|mjs|cjs)(?:$|\?)/, 'js'],
```

Do you mean this?

Reply:
Sorry it was my mistake. The extra `|` operator should have no meaning